### PR TITLE
New package: neomutt-20160404

### PR DESCRIPTION
**Pull request by Vaelatern**

### CODE REVIEW

**Comment by Vaelatern** [comment]
Do I want to add something like
```
provides="mutt-${version}_${revision}"
replaces="mutt>=0"
```
?

**Comment by Duncaen** [comment]
Or add alternatives to `neomutt`, `mutt` and `mutt-kz`.

**Comment by Vaelatern** [comment]
Given the file lists, I am not sure alternatives are the best option. There are 9
manpages in `mutt-kz` and 7 in `mutt`, there are muttrc config files in both, but
Mutt has an extra file in there. There are several similarly named binaries as well.

If it's probably the better way to go, I can start writing alternatives.

**Comment by asergi** [comment]
I really doubt someone wants to install/use all mutt flavors at the same time.
`provides` and `replaces` are enough, imo. Other than that, LGTM.

**Comment by logenkain** [comment]
On Fri, Apr 08, 2016 at 06:07:29AM -0700, Alessio Sergi wrote:
> I really doubt someone wants to install/use all mutt flavors at the same time.
> `provides` and `replaces` are enough, imo. Other than that, LGTM.
> 
> ---
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/voidlinux/void-packages/pull/3981#issuecomment-207424404

Don't underestimate how much I love mutt and want to play with its flavors, but I agree.  I doubt anyone actually wants to use multiple flavors at the same time. I suppose for that we can always use a VM.

**Comment by chneukirchen** [comment]
Could be relevant on a multi-user system...

**Comment by Vaelatern** [comment]
So what should the flavors be renamed? Where things need renaming that is.

**Comment by Vaelatern** [comment]
Okay, going to go with provides and replaces. Should I bother with replaces or just provides?

**Comment by Vaelatern** [comment]
This PR has been updated to have all the mutts using alternative groups. Should sysconfdir be changed, however? Might be cleaner to keep sysconfdir at `/etc` and pollute `/etc` with `Muttrc-$pkgname`

File: srcpkgs/mutt/template [hunk 1]
```diff
@@ -1,32 +1,66 @@
...
-	rm ${DESTDIR}/etc/Muttrc.dist
-	rm ${DESTDIR}/etc/mime.types{,.dist}
-	rm ${DESTDIR}/usr/bin/{flea,muttbug}
-	rm ${DESTDIR}/usr/share/man/man1/{flea,muttbug}.1
+	# provided by mime-types
+	rm ${DESTDIR}/etc/${pkgname}/mime.types{,.dist}
+
+	rm ${DESTDIR}/etc/${pkgname}/Muttrc.dist
+
 	vinstall contrib/gpg.rc 644 etc Muttrc.gpg.dist
```
Comment (Vaelatern) [inline srcpkgs/mutt/template]: Don't know if this should be `vconf` at all. I'm thinking `vsconf` is all that is necessary.

**Comment by Vaelatern** [comment]
Was merged.

**Pull request closed without merging**